recipes/libtiff/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ $(uname) == Linux ]]; then
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+# Install non-lib artifacts to these special directories so we can cleanly delete them later.
+TIFF_BIN="${PREFIX}/tiff-bin"
+TIFF_DOC="${PREFIX}/tiff-doc"
+TIFF_SHARE="${PREFIX}/tiff-share"
+
+mkdir "${TIFF_BIN}" "${TIFF_SHARE}" "${TIFF_DOC}"
+
+# Pass explicit paths to the prefix for each dependency.
+./configure --prefix="${PREFIX}" \
+            --with-zlib-include-dir="${PREFIX}/include" \
+            --with-zlib-lib-dir="${PREFIX}/lib" \
+            --with-jpeg-include-dir="${PREFIX}/include" \
+            --with-jpeg-lib-dir="${PREFIX}/lib" \
+            --with-jbig-include-dir="${PREFIX}/include" \
+            --with-jbig-lib-dir="${PREFIX}/lib" \
+            --with-lzma-include-dir="${PREFIX}/include" \
+            --with-lzma-lib-dir="${PREFIX}/lib" \
+            --bindir="${TIFF_BIN}" \
```

Comment:
Not sure. I just copied the recipe. Let me test here locally and I will get back to you.